client/src/components/manageProjects/eventForm.js

```diff
@@ -11,19 +11,25 @@ const EventForm = ({
 }) => {
   // This creates the clock hours for the form
   const clockHours = createClockHours();
+  const subtypes = [
+    'All Team',
+    'Data Science',
+    'UI/UX Design',
+    'UI/UX Design & Research',
+    'UI/UX Research',
+    'Engineering All',
+    'Engineering Front End',
+    'Engineering Back End',
+    'PM',
+    'PM & Org',
+    'Other',
+  ];
   return (
     <div className="event-form-box">
       {title && <h3 className="event-form-title">{title}</h3>}
       <label className="event-form-label" htmlFor="name">
         Event Name:
-        <input
-          id="name"
-          placeholder="Meeting name..."
-          name="name"
-          value={formValues.name}
-          onChange={handleInputChange}
-          maxLength={30}
-        />
+        <p>{formValues.eventType} {formValues.eventSubtype || ""}</p>
```

Comment:
Use formValues.name here since it will be updated right away when eventType or eventSubtype changes. That way if we change how we're building the name in the future we don't need to update the logic here and the useEffect in createNewEvent